src/main/webapp/app/shared/components/listing-card/listing-card.tsx

```diff
@@ -0,0 +1,38 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import React from 'react';
+import { Row, Col } from 'reactstrap';
+import './listing-card.scss';
+
+export interface IListingCardProps {
+  showActionMenu: boolean;
+  actionMenuHandler?: Function;
+  title?: string;
+}
+
+export class ListingCard extends React.Component<IListingCardProps> {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div className="card-container container">
+        <Row className="align-items-center justify-content-center">
+          <Col xs="10">
+            {this.props.title && this.props.title.hasOwnProperty('value') ? (
```

Comment:
May I know `&& this.props.title.hasOwnProperty('value')` what is this condition checking?
since the `title` can only be typeof `undefined` or `string`

Reply:
Update in f2131ff5e4d6687aed3eb5beb220f1dc9cce08f3